ex1/ex1.js

```diff
@@ -8,25 +8,27 @@
  */
 
 /////////////////////////////////////////////////////////////////////////////
-// getThirdCommaIndex accepts a string and returns the index of the third comma in that string if no comma return -1
-function getThirdCommaIndex(str,searchFor) {
+// Accepts a string and returns the index of the third comma in that string if no comma return -1
+//
+//
+
 
+function getThirdCommaIndex(str,searchFor) {
+        
+	//  i do not know how to explain the use of  -1
```

Comment:
This should have been in a github comment and not in code. (You don't really want this to remain in code for the next time you'll go read this, right?) Please delete.